examples/atl_simple_talker/README

```diff
@@ -0,0 +1,29 @@
+EXAMPLE APPLICATIONS
+
+The 'simple_talker' application illustrates the various steps to publish a stream
+and streaming 1722/61883 audio frames after a listener connects. The audio
+itself is a simple sine wave. It has been tested with other vendors listeners
+at AVNu plug-fests.
```

Comment:
Is the above statement correct? Also please change AVNu to Avnu.

Reply:
That was the plan, but since we didn't make it to plug-fest, I removed the comment. I am currently testing the application with i210 listener, and AQC107 listener. Will add results to the README when completed.